tekton/cronjobs/robocat-dashboard-deploy-latest-cron/cronjob.yaml

```diff
@@ -17,12 +17,8 @@ spec:
               - name: TEKTON_PROJECT
                 value: "dashboard"
               - name: TEKTON_VERSION
-                value: "beta"
+                value: "latest"
               - name: TEKTON_CLUSTER
                 value: "robocat"
-              - name: RELEASE_FILE
-                value: "tekton-dashboard-release-readonly.yaml"
```

Comment:
We should not change the release filename. 
Without this we will deploy the non-readonly dashboard, which would give the world access to the robocat cluster.

Reply:
👍 Thanks for clarifying that, I've put back the original value and added a comment.